README.md

```diff
@@ -50,4 +50,15 @@ Nevertheless, the order of the user stories in each group does not reflect the d
 - As a system administrator, I would like the players to have an enjoyable experience when interacting with the system.
 
 ## Engineering Requirements
-While it might be slightly advantageous if your solution involves technologies/frameworks/programming languages in our tech stack,you are allowed to use any technology/framework/programming language you prefer or are most familiar with. Regardless of which you eventually use, we expect the code to be in high quality.
+While it might be slightly advantageous if your solution involves technologies/frameworks/programming languages in our tech stack:
+- python
```

Comment:
Breaking the sentence like this makes it hard to read. If you want to add these, you probably need to either restructure the sentence or put them somewhere else.

Reply:
Also, could you capitalize them?

Reply:
I personally find list style is easier to read

Reply:
Yes, I agree that a list is easier to read, but not in the middle of a sentence though.